server/index.js

```diff
@@ -0,0 +1,6 @@
+const express = require('express');
+const app = express();
+require('./routes/test.route')(app);
+
+const PORT = 4000;
```

Comment:
we can use process env. If you will not specify port, it will be 4000 by default
const port = process.env.PORT || 4000;

Reply:
LOL. I have deleted process line before commit